templates/assets.go

```diff
@@ -341,7 +341,7 @@ func assetsVpcTargetYml() (*asset, error) {
a4\xdb\x7c\x75\x5f\x7a\x7e\xaf\x94\xbc\xb5\xe7\x20\xcd\xcb\x59\x73\xee\xb8\x89\x39\xb0\x1f\x21\x41\xad\xe3\x14\x32\x55\x96\x99\x2b\x7a\x29\xad\x4d\xa2\x76\x6a\x92\x52\x4b\xa6\x73\x93\x4c\x4e\xcb\x24\xaa\xec\x67\x8b\x3d\x4d\x32\xed\xd4\x24\xa5\x8e\x55\xe7\x26\x51\x4f\xcb\x24\x53\xc1\x24\xc7\x2f\x62\xf7\x57\x48\x50\xa7\xf3\x52\xb6\x8c\xd3\x4e\xf5\x6c\x76\x78\x14\xb3\x4b\x1c\xe7\x4b\xd1\xba\xc3\x72\xb6\x5d\x96\x29\x64\x4b\xf9\x63\xcd\x61\x2a\xc5\xbe\xde\x14\xac\x8a\xd9\x5b\x14\x54\xf7\x54\x70\xda\x9b\x82\x55\x11\x70\x8b\x82\xd3\x96\x0a\x1e\xff\x0c\xd3\x69\x94\xe8\xa7\x4a\x2f\x83\x77\x88\xf3\xcc\xe1\x4a\xf5\x22\x5a\x3f\x2c\xff\x34\x00\x93\x9d\x6d\x26\xff\x3a\x21\xc4\xf8\x49\x79\xee\x6f\x90\x8b\x02\xe8\xd0\xa5\x4e\x03\xba\xe9\x91\xcf\xce\x93\xb1\x7a\xbe\x2b\x7a\x3f\x2c\xff\x74\x00\x54\xf7\xdb\x7b\xbd\x22\xb8\x5a\x5d\x15\x8f\xd8\xc7\x84\xee\xfc\x20\x7b\x6f\x6b\x2f\xba\x1a\xca\xfc\x2f\x56\xe9\x40\xa9\xec\xe9\xbc\x4f\x71\xd0\xea\xac\xa9\x85\x4a\xc5\x4f\xcf\x6a\xee\x59\xa3\xb5\x55\x73\xda\xb3\x9a\x7b\x56\x6a\xcd\xd4\xe4\x3d\xaa\xf9\xd7\x8f\xf9\xf6\x54\xf9\x26\x50\xe3\x33\xb7\xf8\xe8\x5e\xc7\xed\xba\xbb\x44\xd2\x93\xf6\x2b\x7d\x0d\xfe\x2b\xde\x16\x79\xc3\x7e\x3a\x7a\x13\x1f\xc1\xc1\xff\x5a\x1e\xc2\x3b\x6c\x55\x49\x24\x1c\xc6\x9f\x87\xf4\x53\x5c\xab\x0e\x53\x49\x5b\x94\xab\x8e\xc1\x7a\x59\xe5\xfb\x55\x6d\xec\xb7\x5f\x07\xab\xee\x86\x56\x9f\xf6\xeb\xb0\xaf\xd5\x8b\xfd\xd4\xc2\x45\xfa\xf6\xf6\xdb\xaf\xdd\x55\x77\xef\xad\x4f\xfb\x75\xd8\x04\xeb\xc5\x7e\xd3\xd8\x7e\x47\x6f\x90\xf5\xa1\x6c\xac\xea\x2e\xcd\x33\x61\xb3\x0a\xa2\xb1\xa9\xed\x57\x2c\x1a\x75\xde\x72\xd8\x73\xb2\xae\x7e\x44\x16\x73\xd2\xc1\x7a\x36\x0d\x32\x68\xa1\x08\x28\x3d\xd1\x08\x18\x31\x40\xf7\xa1\x4c\xc3\xc2\xad\xf4\xc4\xae\xca\x1c\xae\x9b\xd6\x20\xb6\x56\x2b\x53\xd3\x43\x93\x28\x73\xf4\xce\x59\x7f\xe1\x83\x9f\x3a\x93\x2b\xba\x87\x3d\x74\xe6\xa0\xed\xfd\xbc\x29\xb9\x6f\xd8\xa2\x5d\x94\x61\x76\x45\xc8\x81\x7b\x6b\x0d\xf0\x7a\x77\xe4\xde\xd0\xef\x45\xa0\xe2\x81\x3c\x4a\xe1\xd1\x61\x7a\x7f\x64\x98\xce\xcf\xa7\x05\x9c\xf8\xc8\xdc\x31\xfa\x69\x6c\x37\x00\xe9\x54\x7b\xda\x62\x86\x3d\x70\x23\xa0\x93\xf4\xdf\x00\x7d\x31\xcf\xf6\xa7\x52\xfb\x22\x60\x37\x95\x0e\xdd\xae\xe9\xa4\x14\xd8\xaa\xd2\xe0\x7b\x44\xfc\x88\xb0\x25\xe3\x84\xce\x2b\x65\xe1\x8a\xfe\xcd\x06\x01\xdb\x02\xde\x1a\x90\x0d\xe2\x17\x79\xb3\xdc\x2b\xa6\xff\xf9\x33\xbb\xd0\x9e\x30\x8f\xdf\xb6\x90\x27\x67\xc6\x4c\xf6\xfe\x48\xfe\x4d\xd1\x65\xc2\x96\xbf\xb6\x00\x36\x5e\x48\x72\xfc\xf9\xad\x66\x61\x9d\xb3\xe4\xe4\x28\x41\x35\x7b\x69\x21\x45\x4b\xb7\xc2\x0a\xa5\x91\x63\x80\x90\xd1\xd8\x56\x28\x30\xe5\xfa\xd1\xb3\xe2\x7f\x3c\x1b\x8b\x67\x49\xe5\x4d\xe6\xe0\xa3\xf4\x53\x72\xb2\xd4\xd7\x9a\x30\x46\x47\x65\xee\x98\xcd\x56\x39\x5f\x6e\x9e\x21\xbb\xf0\x98\x68\x3b\x73\x54\xb7\x70\x94\xce\xef\xc5\x71\xba\x85\xa3\x74\xbe\xc4\x71\x87\x9d\x26\x1a\xba\xd4\x76\xad\x31\xbe\xa9\xf6\x60\xfc\xaa\x5e\x79\x4e\xfb\x46\x44\xbb\x1b\xa5\xaa\x81\x5d\xcb\xbb\xa3\x0d\x51\xd5\x55\xae\xe5\xdd\xf9\xd6\x28\xed\x85\xc1\xe0\xaf\x00\x00\x00\xff\xff\x2b\x3f\xea\xa5\xf6\x47\x00\x00")
```

Comment:
@cplee What is this _assetsVpcYml byte array?  It seems to the encoding of something else, but it's not clear to me of what!  The first 10 bytes don't look like yml content, which is what the variable name suggests.

Reply:
This file is generated based on the static yml files in the source repo